source/whats-new.txt

```diff
@@ -53,6 +54,43 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.3:
+
+What's New in 6.3
+-----------------
+
+The {+driver-short+} v6.3 release includes the following features:
+
+- Adds the ``serverMonitoringMode`` client option to control the
+  behavior of the monitoring connection between each node in a topology.
+  This option takes a value of ``auto`` (default), ``poll``, or
+  ``stream``. When the setting is ``auto``, the environment the driver
+  runs in determines the monitoring mode. Function-as-a-service (FaaS)
+  environments prefer the polling mode and all others prefer the
+  streaming mode.
```

Comment:
S: "prefer" doesn't seem quite right and is pretty [anthropomorphic](https://www.mongodb.com/docs/meta/style-guide/writing/use-effective-verbs/#don-t-humanize-inanimate-objects)

```suggestion
  runs in determines the monitoring mode. The driver uses polling
  mode in function-as-a-service (FaaS)
  environments and streaming mode everywhere else.
```